### ISSUE TITLE
scalinthealth-common config file not being created

### ISSUE THREAD

**Issue by CoryGuy** [SilentChaos512/ScalingHealth#426]
## Versions

- **Scaling Health**: 1.19.2-7.0.0+6.jar
- **Silent Lib**: 1.19.2-7.0.3.jar
- **Forge**: 43.1.57
- **Modpack**: N/A 
- **Optifine Installed**: No 

## Expected Behavior

The common.toml config file is not being created, and I can't decrease my player's starting health or leveling curve

## Actual Behavior

Only scalinghealth-client.toml is being created.

**Comment by dsnidr** [comment]
Same here, I could not get any configs to generate other than the client config.

**Comment by domshi** [comment]
i too am having the issue where the config file for modifying heart maximum is not being created therefore i cannot change the environmental difficulty as there can be players with too much health

**Comment by dsnidr** [comment]
I ended up finding the file. It generates in the world's config folder, not the config folder at the root of the server.

**Comment by domshi** [comment]
> I ended up finding the file. It generates in the world's config folder, not the config folder at the root of the server.

i have a file there but is does not show config for number of hearts

**Comment by togera13** [comment]
1.19.2-7.0.0 Also seems to be missing the Min/Max health options (Also for min/max damage boost).
Both Client & the save's config folder do not contain any configs with any options for these.

Idk if I should have made this a new post, I got distracted by the post above mine.

**Comment by AngellusMortis** [comment]
Yeah, seeing `config/scalinghealth-client.toml` and `world/serverconfig/scalinghealth-server.toml`, but not seeing any of the common or dimension configs as [documented on the wiki](https://github.com/SilentChaos512/ScalingHealth/wiki/Configuring-the-Mod-(1.13-)#files).

**Comment by Survivor214** [comment]
Found the client and server config as well, but the same missing options in version 1.20.1-8.0.2+9
